sdk/src/driftClient.ts

```diff
@@ -5882,10 +5883,15 @@ export class DriftClient {
 	public encodeSwiftOrderParamsMessage(
 		orderParamsMessage: SwiftOrderParamsMessage
 	): Buffer {
-		return this.program.coder.types.encode(
-			'SwiftOrderParamsMessage',
-			orderParamsMessage
-		);
+		const anchorIxName = 'global' + ':' + 'swiftOrderMessageParams';
+		const prefix = Buffer.from(hash(anchorIxName).slice(0, 8));
```

Comment:
Can we set prefix = `Buffer.from(hash('global:swiftOrderMessageParams')).slice(0,8)` instead?

Right now, this is adding a 14 byte discriminator 🙃 

```sh
$ node
> const { hash } = await import("@coral-xyz/anchor/dist/cjs/utils/sha256.js");
undefined
> Buffer.from(hash('global:swiftOrderMessageParams').slice(0,8)).length
14
> Buffer.from(hash('global:swiftOrderMessageParams')).slice(0,8).length
8
```